apps/frontend-manage/src/components/sharing/usePermissionLevelChange.ts

```
@@ -97,9 +95,7 @@ function usePermissionLevelChange({
         ],
       })
 
-      if (
-        typeof res.data?.changePermissionLevel?.permissionId !== 'undefined'
-      ) {
+      if (typeof res.data?.changePermissionLevel) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)